web/frontend/src/components/base.vue

```diff
@@ -252,6 +260,9 @@ onClickOutside($$(root), () => {
 onMounted(() => {
   window.addEventListener('visibilitychange', handleVisibilityChange);
 
+  // Safety measure for system prompts, etc.
```

Comment:
This actually happened to me while testing. Somehow I triggered my Macbook to ask me if I wanted dictation, and the motor just kept going. This was before the fix however so maybe not needed. Thoughts?

Reply:
Seems like a good redundancy if nothing else